pkgs/development/python-modules/openai/default.nix

```diff
@@ -29,29 +40,28 @@ buildPythonPackage rec {
     owner = "openai";
     repo = "openai-python";
     rev = "v${version}";
-    hash = "sha256-bwv7lpdDYlk+y3KBjv7cSvaGr3v02riNCUfPFh6yv1I=";
+    hash = "sha256-dq0as07a5gSFbwLFjxKHBZFB51xjAPqs1FFrp0ScXBY=";
   };
 
   propagatedBuildInputs = [
-    numpy
-    openpyxl
-    pandas
-    pandas-stubs
+    aiohttp
     requests
     tqdm
+  ] ++ lib.optionals (pythonOlder "3.8") [
     typing-extensions
   ];
 
   passthru.optional-dependencies = {
```

Comment:
```suggestion
  passthru.optional-dependencies = rec {
```

Reply:
Can't use `rec` here cause it because `wandb` from `passthru.optional-dependencies` to shadow `wandb` (the Python package). So I used `passthru.optional-dependencies.datalib` since there is already a `rec` on for on the set of the package definition.